intg/src/main/java/org/apache/atlas/model/instance/AtlasRelationshipHeader.java

```diff
@@ -80,6 +84,10 @@ public AtlasRelationshipHeader(AtlasRelationship relationship) {
             case DELETED:
                 setStatus(AtlasEntity.Status.DELETED);
                 break;
+
+            case PURGE_DUE_TO_ENTITY:
```

Comment:
can you rename to be more specific `PURGE_DUE_TO_ENTITY` seems incomplete.

Reply:
Done